release/test_release_gold.sh

```diff
@@ -66,6 +76,18 @@ function test_prepare_next_release_branch {
 		"2024.Q1.13"
 }
 
+function test_reference_new_releases {
+	lc_cd "test-dependencies/actual.liferay-jenkins-ee.build.properties"
```

Comment:
@kiwm, can we use just `actual.build.properties` and `expected.build.properties` instead for simplicity?